cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -572,14 +585,16 @@ public TwillController start(long timeout, TimeUnit timeoutUnit) {
         }
       }
 
+      Map<String, String> cConf = masterEnvContext.getConfigurations();
```

Comment:
don't need this part where we pass the cconf down anymore

Reply:
Fixed